src/overlays/actors/ovl_En_Pm/z_en_pm.c

```diff
@@ -1344,7 +1344,7 @@ s32 func_80AF9BF8(EnPm* this, PlayState* play, ScheduleOutput* scheduleOutput) {
 }
 
 s32 func_80AF9D04(EnPm* this, PlayState* play) {
-    EnDoor* door = (EnDoor*)func_80AF7D60(play, this->unk_258);
+    EnDoor* door = (EnDoor*)EnPm_FindDoorSchedule(play, this->unk_258);
```

Comment:
```suggestion
    EnDoor* door = EnPm_FindDoorSchedule(play, this->unk_258);
```
Cast doesn't seem necessary?